plugin/generator.go

```diff
@@ -105,6 +117,19 @@ func (g *CaddyfileGenerator) GenerateCaddyFile() []byte {
 
 	directives := map[string]*directiveData{}
 
+	if g.caddyFilePath != "" {
+		dat, err := ioutil.ReadFile(g.caddyFilePath)
+		if err != nil {
+			if _, err := buffer.Write(dat) {
```

Comment:
This line can't compile.
Do you mean `if _, err := buffer.Write(dat); err {}` ?

Reply:
Yes, thanks! I made a PR to solve it #82